components/icu/src/lib.rs

```diff
@@ -169,7 +169,7 @@ pub mod decimal {
     //! assert_eq!("2,000.50", fdf.format(&fixed_decimal).writeable_to_string());
     //! ```
     //!
-    //! [`FixedDecimal`]: fixed_decimal::FixedDecimal
+    //! [`FixedDecimal`]: decimal::FixedDecimal
```

Comment:
Question: How does this work?  `FixedDecimal` is in the `fixed_decimal` crate, not the `icu_decimal` crate.

Reply:
Ah for some reason I thought it was a reexport